package.json

```diff
@@ -14,7 +14,7 @@
     "lodash": "^4.1.0",
     "mime": "^1.3.4",
     "mout": "^0.11.1",
-    "needle": "^0.11.0",
+    "needle": "^1.4.2",
```

Comment:
@chrisjhoughton this is for the fix for that file size issue, but its quite a large jump for needle version, perhaps its worth a larger bump in falafel version?